baseclasses/solvers/pyAero_solver.py

```diff
@@ -94,6 +95,16 @@ def setDVGeo(self, DVGeo, pointSetKwargs=None):
         pointSetKwargs : dict
             Keyword arguments to be passed to the DVGeo addPointSet call.
             Useful for DVGeometryMulti, specifying FFD projection tolerances, etc.
+            These arguments are used for all point sets added by this solver.
+
+        customPointSetFamilies : dict of dicts
+            Keyword arguments to be passed to the DVGeo addPointSet call for each surface family,
+            specified by the keys. The surface families need to be all part of the designSurfaceFamily.
+            Useful for DVGeometryMulti, specifying FFD projection tolerances, etc.
```

Comment:
I feel like the three lines above do not emphasize the main purpose of this argument, which is to have subsets of the CFD surface as separate point sets. I would rephrase this to first talk about what the keys are and what they do, then talk about how the values are dictionaries of keyword arguments.